hypothesis-python/tests/nocover/test_strategy_state.py

```diff
@@ -72,47 +72,6 @@ def set_database(self):
         self.teardown()
         self.database = ExampleDatabase()
 
-    @rule(strat=strategies, r=integers(), max_shrinks=integers(0, 100))
```

Comment:
Why did these go away?

Reply:
Because they simply didn't work with `min_satisfying_examples > 0`!  

I'd love to know why, but after spending 6+ hours digging I'm not sure they were really working (as in, finding any examples at all) before.  It's practically impossible to work out *why* a state machine fails with `Unsatisfiable`, BTW :sob:

Reply:
Thanks past-@DRMacIver. You leave such fun things in the code base for us to discover.